docs/getting_started/install_manually.md

```diff
@@ -335,160 +337,142 @@ See [Set up directory permissions](set_up_directory_permissions.md) for more inf
 
-Composer is a dependency manager that allows you to install packages directly in the project. It is also checking all packages' versions on a regular basis to make sure they are up-to-date and to avoid inconsistencies.
+You can also clone one of the [repositories from GitHub](#available-distributions).
 
 ``` bash
-curl -sS https://getcomposer.org/installer | php
-php -d memory_limit=-1 composer.phar
+git clone https://github.com/ezsystems/ezplatform.git
 ```
 
-### 5. Create a new database for eZ Platform
+You can check out a tag, or use the `master` branch if you are interested in working with the latest version.
 
-Create new database (you can substitute `ez1` with the database name you want to use):
+!!! tip
 
-``` bash
-mysql -uroot -ppassword -e "CREATE DATABASE ez1"
-```
+    You can use any other folder name in place of `ezplatform` in the examples above.
+    You'll point your virtual host to this folder to use as its root.
 
-### 6. Install additional requirements for eZ Platform
 
-a. Install PEAR/PECL extension:
+### Install Composer globally
 
-``` bash
-cd c:\program files\php\php.ini
-curl -O https://pear.php.net/go-pear.phar
-php -d detect_unicode=0 go-pear.phar
-php install-pear-nozlib.phar
-pear channel-update pear.php.net
-pecl channel-update pecl.php.net
-pear upgrade-all
-pear config-set auto_discover 1
-```
+Download and run [Composer-Setup.exe](https://getcomposer.org/download/) - it will install the latest composer version whenever it is executed.
+
+#### Install dependencies with Composer
 
-b. Edit `php.ini` and add following line:
+From the folder into which you downloaded eZ Platform, run:
 
 ``` bash
-extension=intl.so
+composer install
 ```
 
-c. Enable opcache extension for PHP (suggested, but not required) by adding:
+Once the installer gets to the point that it creates `app/config/parameters.yml`, you will be presented with a few decisions:
 
-``` bash
-zend_extension=opcache.so
-```
+1. Choose a [secret](http://symfony.com/doc/current/reference/configuration/framework.html#secret); it should be a random string, made up of up to 32 characters, numbers, and symbols. This is used by Symfony when generating [CSRF tokens](http://symfony.com/doc/current/book/forms.html#forms-csrf), [encrypting cookies](http://symfony.com/doc/current/cookbook/security/remember_me.html), and for creating signed URIs when using [ESI (Edge Side Includes)](http://symfony.com/doc/current/book/http_cache.html#edge-side-includes).
+2. You can accept the default options for `database_driver`, `database_host` and `database_port`
+3. For `database_name` and `database_user`, replace them if you customized those values during configuration.
+4. If you set a password for your database user, enter it when prompted for `database_password`.
+
+The installer should continue once you've completed this manual portion of the installation process.
 
-### 7. Install eZ Platform
+### Create a new database for eZ Platform - optional
```

Comment:
Remember to remove "optional" for v1 - database is created automatically in v2 only

Reply:
ok